python/lsst/sconsUtils/scripts.py

```diff
@@ -481,9 +507,31 @@ def s(l):
             swigMods.extend(
                 state.env.SwigLoadableModule("_" + name, src, LIBS=state.env.getLibs("main python"))
             )
+
+        # Warn about insisting that a test in pySingles starts with test_ and therefore
+        # might be automatically discovered by pytest. This warning could only be issued
```

Comment:
Don't know. Do you think it should only be issued if auto test discovery is enabled? I think I'd prefer the warning regardless so that we don't get hit with renaming the file again when auto test discovery is turned on. If we don't warn we are going to get problems later when people think they can just run `pytest` in the root directory.